### DISCUSSION TITLE
استفاده از BIP47 روی بیت کوین برای کمک مالی ناشناس به اعتصاب کنندگان

### DISCUSSION THREAD

**Discussion by FreedomFighterOfIran** [WeAreMahsaAmini/Core#482] [Ideas]
سلام
روی بیت کوین یه تکنولوژی به اسم BIP47 وجود داره که با کمک اون میشه تراکنشی رو روی بیت کوین انجام داد که فقط طرفین تراکنش ازش مطلع میشن. با استفاده از این تکنولوژی، مردم میتونن برای اعتصاب کنندگان به طور ناشناس بیت کوین ارسال کنند و بعد اعتصاب کنندگان اون بیت کوین ها رو تو حساب صرافی خودشون یا نزدیکانشون نقد کنند و استفاده کنند.

شیوه استفاده ش هم به این صورته: ابتدا باید این ولت بیت کوین رو دانلود کنید
https://play.google.com/store/apps/details?id=com.samourai.wallet&hl=en_GB&gl=US
و بعد مراحلش رو انجام بدید تا ولت بیت کوینتون ساخته بشه. بعد روی + پایین صفحه کلیک کنید و روی PayNyms بزنید. یک لینک پرداخت براتون ساخته میشه مثل این:
http://paynym.is/+youngrain628
بعد سایرین میتونن با استفاده از این لینک پرداخت تو همین ولت، برای شما به صورت ناشناس بیت کوین ارسال کنن

**Comment by IJustWantSomeFreedom** [comment]
خب دو تا سوال پیش میاد:
- من از کجا بدونم که اون فردی که حساب بیت کوینش رو گذاشته واقعا اعتصاب کرده؟
- خیلی از افراد اینجوری دیده نمیشن. مثلا یه فروشنده ساده ای که اعتصاب کرده، چطوری مردم باید خبر دار بشن و به حسابش بیت کوین بریزن؟

**Reply by moh-sanati** [reply]
بنظرم یک تیم باید اطلاعات کارکنان و یا کارگران اعتصاب کنندگان جمع آوری کنند و اینکه باید با اولویتبندی تاثیر گذاری بیشتر انجام شود

**Reply by FreedomFighterOfIran** [reply]
آره یک تیم اعتبارسنجی نیاز داریم

**Comment by moh-sanati** [comment]
انتقال بر بستر بیتکوین هزینه بر است 
برای انتقال همین که مستقیم به کیف پول افراد انتقال داده شود با برخی از نکات ساده هویت افراد مشخص نخواهد شد.
@FreedomFighterOfIran

**Reply by FreedomFighterOfIran** [reply]
اگر قرار باشه اندازه حقوق یک ماه فرد مثلا بهش واریز کنیم، به فی انتقال می صرفه. و خب طبیعتا privacyای که به افراد میده هزینه داره
و اگر قرار باشه تو یه وبسایت عمومی قرار داده بشن ولت ها برای کمک، خب اون ولت شناسایی میشه نهایتا وقتی میخواد پول رو به اکسچنج واریز کنه. اما با payment code های bip47 این خطر وجود نداره

**Comment by vovak1401** [comment]
اونا می تونن اینجا عضو بشن و یه جورایی احراز هویت بکنن?

**Reply by FreedomFighterOfIran** [reply]
بنظرم یه تیم باید این کار رو انجام بدن

**Comment by FreedomFighterOfIran** [comment]
من این discussion رو 20 روز پیش شروع کردم ولی دیدم روش واکنشی نیومده بود دیگه بعد پیگیری نکردم
اما بعد چند روز دیدم ظاهرا یه سری بحث ها روش شده و بنظرم ارزش فکر و ادامه داره

اگر ایده ای دارید برای این که چطور بفهمیم آدرس های پرداختی که داره گذاشته میشه واقعا مربوط به یک سری اعتصاب کننده هست بگید.
یا اگر خودتون شبکه ای از اعتصاب کننده ها میشناسید

**Reply by redflake** [reply]
had the same idea. Certainly there are some mechanisms to check who is approved for payment.

First we should not be too perfectionist about it
Not all arrangements can be done 100% correctly. We are in a situation where it is worthwhile if 80% of the verifications are done correctly.
So then we have multiple different mechanisms:
1- The striker can reveal his identity or get discovered by journalists.
* You are asking about privacy and anonymity?
* Remember that he is on strike and not doing his job, so his name is already known to the regime.

2- You can verify the striker by other verified users! In many cases strikers know each other and can validate Themesleves, but first you need to verify the right ones
3- by just showing up and sending pictures and documents to prove they are striking! They are already under pressure, it would help them to earn money, it is not as risky as leaving a job and having nothing to eat.

**Reply by vovak1401** [reply]
گروهی به نام ققنوس به شدت فعال وجود داره، و به فعالین و مبارزان کمک می کنه، می تونی با رئیس گروه تماس بگیری و نحوه کمک مالی رو بررسی کنیم
https://t.me/+Y5A4cQFfNZI5MmNi
